src/cli/add.rs

```diff
@@ -376,7 +371,11 @@ async fn determine_latest_versions(
 ) -> miette::Result<Vec<Version>> {
     // Get platforms to search for including NoArch
     let platforms = if platforms.is_empty() {
-        let mut temp = project.platforms().into_iter().collect_vec();
+        let mut temp = project
```

Comment:
shouldn't latest versions be determined by environment?

Reply:
Yes they should, the determine function is heavily outdated. @baszalmstra is going to work on it next week.